endpoints/openrtb2/sample-requests/video/video_invalid_sample.json

```diff
@@ -1,68 +1,69 @@
 {
```

Comment:
I normalized the format of the sample json files. I removed the no longer existing `accountid` field, moved the no longer valid `gdpr` section (it's uses the standard IAB locations now), and reformatted the `user ` section to match IAB spec from an earlier customized `user` object. .. oh, and ran it through a json "prettifier".